cli/run.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/alibaba/pouch/pkg/reference"
+	"github.com/spf13/cobra"
+)
+
+// runDescription is used to describe run command in detail and auto generate command doc.
+var runDescription = "Create a container object in Pouchd, and start the container. " +
+	"This is useful when you just want to use one command to start a container. " +
+	"Also, you can specify --rm to delete it after the container quits. "
+
+// RunCommand use to implement 'run' command, it creates and starts a container.
+type RunCommand struct {
+	baseCommand
+	container
+	attach bool
+	stdin  bool
+	rm     bool
+}
+
+// Init initialize run command.
+func (rc *RunCommand) Init(c *Cli) {
+	rc.cli = c
+	rc.cmd = &cobra.Command{
+		Use:   "run a container",
+		Short: "Create a new container and start it",
+		Long:  runDescription,
+		Args:  cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return rc.runRun(args)
+		},
+		Example: runExample(),
+	}
+	rc.addFlags()
+}
+
+// addFlags adds flags for specific command.
+func (rc *RunCommand) addFlags() {
+	flagSet := rc.cmd.Flags()
+	flagSet.StringVar(&rc.name, "name", "", "Specify name of container")
```

Comment:
Let us add `flagSet.SetInterspersed(false)` here to make args not in scope of flag. For example: `pouch run image ls -l` will be OK if we set `flagSet.SetInterspersed(false)`. Otherwise `-l` we be an error cause, because `-l` will be handled as flag.

Reply:
I think it's not good for the flags can't reuse the "create" and "start" command's flags. 

WDYT @HusterWan